js/ui/layout.js

```diff
@@ -352,8 +352,7 @@ var LayoutManager = GObject.registerClass({
             this.bottomMonitor = this.monitors[this.bottomIndex];
 
             if (this._pendingLoadBackground) {
-                this._loadBackground();
```

Comment:
I believe these changes will need to be in a quilt patch

Reply:
Changes added as a quilt patch